pom.xml

```diff
@@ -20,6 +20,29 @@
     </properties>
 
     <dependencies>
+        <!-- https://mvnrepository.com/artifact/io.jsonwebtoken/jjwt-api -->
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt-api</artifactId>
+            <version>0.11.2</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/io.jsonwebtoken/jjwt-impl -->
+        <dependency>
```

Comment:
There are some marked vulnerabilities marked for this dependency. https://mvnrepository.com/artifact/io.jsonwebtoken/jjwt-impl/0.11.2. Please take a look